src/v2/openapi.yaml

```diff
@@ -0,0 +1,740 @@
+          type: string
+          enum:
+            - low
+            - medium
+            - high
+            - critical
+        title:
+          type: string
+          example: SQL Injection
+        id:
+          type: string
+          example: SNYK-PYTHON-DJANGO-40027
+        url:
+          type: string
+          example: https://snyk.io/vuln/SNYK-PYTHON-DJANGO-4002
+      description: Limited private vulnerability details for free tier users
+
+    PremiumVulnerabilityFields:
+      title: PremiumVulnerabilityFields
+      type: object
+      properties:
+        malicious:
+          type: boolean
+          default: true
+        patch_exists:
+          type: boolean
+          default: false
+          example: true
+        fixable:
+          type: boolean
+          default: false
+          example: true
+        exploit:
+          type: string
+          enum:
+            - High
+            - Functional
+            - Proof of Concept
+            - Unproven
+            - Not Defined
+          example: High
+        description:
+          type: string
+          example: |
+            "## Overview\r\n[`django`](https://pypi.python.org/pypi/django) is a high-level Python Web framework that encourages rapid development and clean, pragmatic design.\r\n\r\nAffected versions of this package expose sensitive information due to not properly restricting the use of a query string that performs certain object filtering. An attacker may obtain sensitive information via a series of requests containing regular expressions, as demonstrated by a `created_by__password__regex` parameter.\r\n\r\n## References\r\n- [Django Vulnerability Description](http://www.djangoproject.com/weblog/2010/dec/22/security/)\r\n- [GitHub Commit](https://github.com/django/django/commit/732198ed5c)\r\n- [Redhat Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=665373)\r\n- [Openwall](http://www.openwall.com/lists/oss-security/2011/01/03/5)\r\n- [CVE](https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2010-4534)\r\n"
+        fixed_in:
+          type: array
+          items:
+            type: string
+          example:
+            - 2.6.7
+            - 2.7.5
+            - 2.8.0
+        references:
+          type: array
+          items:
+            type: object
+            properties:
+              title:
+                type: string
+                example: Redhat Security Advisory
+              url:
+                type: string
+                example: https://access.redhat.com/security/cve/CVE-2013-2186
+      description: |
+        Extended Vulnerability details, available only to registered users
+
+    Package:
+      title: Package
+      type: object
+      required: 
+        - name
+        - version
+      properties:
+        name:
+          type: string
+          example: django
+        version:
+          type: string
+          example: 1.2.1
+      description: Package details
+
+    PackageWithTransitives:
```

Comment:
This should be included within the package itself as a deps array.

Reply:
fixed it.